server/base/src/main/java/org/apache/accumulo/server/fs/FileTypePrefix.java

```diff
@@ -18,27 +18,18 @@
  */
 package org.apache.accumulo.server.fs;
 
-import java.util.EnumSet;
-import java.util.HashSet;
 import java.util.Objects;
-import java.util.Set;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 import com.google.common.base.Preconditions;
 
 public enum FileTypePrefix {
 
-  ALL("*"),
-  FLUSH("F"),
+  FLUSH(
+      "F"),
```

Comment:
I have no idea why the formatter is putting a new line here. I removed it locally, and it put it back.

Reply:
I'm not sure why this happens. There may be some obscure Eclipse formatter config for it, but it can be prevented by adding a comment after, or separating enums with blank lines (perhaps with a javadoc for each, for example).